controllers/sandbox_controller.go

```diff
@@ -302,7 +323,7 @@ func (r *SandboxReconciler) reconcilePod(ctx context.Context, sandbox *sandboxv1
 	if err := ctrl.SetControllerReference(sandbox, pod, r.Scheme); err != nil {
 		return nil, fmt.Errorf("SetControllerReference for Pod failed: %w", err)
 	}
-	err := r.Create(ctx, pod, client.FieldOwner("sandbox-controller"))
+	err = r.Create(ctx, pod, client.FieldOwner("sandbox-controller"))
```

Comment:
Nit: I do think it's better to combine the err onto one line with the if statement

Reply:
changed it to use if err statements.